database/migrations/2020_01_08_043027_create_blogs_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBlogsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     */
+    public function up()
+    {
+        Schema::create('blogs', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('user_id');
+            $table->string('title');
+            $table->text('content');
+            $table->string('image_urls');
+            $table->timestamps();
+            $table->softDeletes();
```

Comment:
where is the usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=41579321 Column Modifiers ?

Reply:
Dạ em đã chỉnh edit function up(): Em thêm 5 trường là: $table->bigInteger('user_id');
            $table->string('title');
            $table->text('content');
            $table->string('image_urls');